src/app/helpers/tests/testHelpers.js

```diff
@@ -30,3 +30,12 @@ export const isNull = (title, component) => {
     expect(tree).toBeNull();
   });
 };
+
+export const validateHttpHeader = (headers, headerKey, expectedHeaderValue) => {
```

Comment:
This probably doesn't need to be its own helper because it's not used anywhere else but server.